Code/autopkglib/StopProcessingIf.py

```diff
@@ -47,14 +55,51 @@ class StopProcessingIf(Processor):
     }
 
     def predicate_evaluates_as_true(self, predicate_string):
-        """Evaluates predicate against our environment dictionary"""
-        try:
-            predicate = NSPredicate.predicateWithFormat_(predicate_string)
-        except Exception as err:
-            raise ProcessorError(f"Predicate error for '{predicate_string}': {err}")
-
-        result = predicate.evaluateWithObject_(self.env)
-        self.output(f"({predicate_string}) is {result}")
+        if is_mac():
+            """Evaluates predicate against our environment dictionary"""
+            try:
+                predicate = NSPredicate.predicateWithFormat_(predicate_string)
+            except Exception as err:
+                raise ProcessorError(f"Predicate error for '{predicate_string}': {err}")
+
+            result = predicate.evaluateWithObject_(self.env)
+            self.output(f"({predicate_string}) is {result}")
+            return result
+        elif is_windows():  # Added on Windows version
+            result = False
+            try:
+                # result = eval(predicate_string)
+                splitted_ops = predicate_string.split()
```

Comment:
I think this is assuming that predicates have a space between the left side, the right side, and the operator(s) but I don't think that is always the case. Is `download_changed!=True` a valid predicate? I don't actually have enough experience with them to know for sure.

Reply:
This isn't a true replacement for `StopProcessingIf`'s use of predicates, but it does work the same way cross platform: https://github.com/jgstew/jgstew-recipes/blob/main/SharedProcessors/StopProcessingIfDownloadUnchanged.py